crates/sovereign-sdk/full-node/sov-stf-runner/src/runner.rs

```diff
@@ -300,6 +306,10 @@ where
                 .da_service
                 .extract_relevant_blobs(&filtered_block)
                 .into_iter()
+                .filter(|tx| {
```

Comment:
parse first

- if seq commitment check for sequencer commitment
- if proof check for zk proof